macros/shared

```diff
@@ -152,6 +152,44 @@ CROSS_COMPILATION_CONF_EOF\
 %cross_meson_install \
   %ninja_install -C %{_cross_vpath_builddir}
 
+%_cross_cmake_toolchain_conf %{_cross_vpath_srcdir}/toolchain.%{_cross_target}.cmake
+
+%write_cross_cmake_toolchain_conf \
+cat <<CROSS_CMAKE_TOOLCHAIN_EOF > %_cross_cmake_toolchain_conf 2>/dev/null || :\
+set(CMAKE_SYSTEM_NAME Linux)\
+set(CMAKE_SYSTEM_PROCESSOR %{_cross_arch})\
+set(triple %{_cross_triple})\
+set(CMAKE_SYSROOT %{_cross_rootdir})\
+set(CMAKE_C_COMPILER %{_cross_target}-gcc)\
+set(CMAKE_C_COMPILER_TARGET %_cross_target)\
+set(CMAKE_CXX_COMPILER %{_cross_target}-g++)\
+set(CMAKE_CXX_COMPILER_TARGET %_cross_target)\
+set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)\
+set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)\
+set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)\
+set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)\
+CROSS_CMAKE_TOOLCHAIN_EOF\
+%{nil}
+
+%_cross_cmake_toolchain_conf_static %{_cross_vpath_srcdir}/toolchain.%{_cross_triple}-musl.static.cmake
+
+%write_cross_cmake_toolchain_conf_static \
+cat <<CROSS_CMAKE_TOOLCHAIN_EOF > %_cross_cmake_toolchain_conf_static 2>/dev/null || :\
+set(CMAKE_SYSTEM_NAME Linux)\
+set(CMAKE_SYSTEM_PROCESSOR %{_cross_arch})\
+set(triple %{_cross_triple})\
+set(CMAKE_SYSROOT /%{_cross_triple}-musl/sys-root)\
+set(CMAKE_C_COMPILER %{_cross_triple}-musl-gcc)\
+set(CMAKE_C_COMPILER_TARGET %{_cross_triple}-musl)\
+set(CMAKE_CXX_COMPILER %{_cross_triple}-musl-g++)\
+set(CMAKE_CXX_COMPILER_TARGET %{_cross_triple}-musl)\
+set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)\
+set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)\
+set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)\
+set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)\
+CROSS_CMAKE_TOOLCHAIN_EOF\
```

Comment:
I'd still like to see `%cross_cmake` updated to reflect this.

Reply:
Yes, that was the last set of changes, I didn't want to play wack-a-mole with `docker-init` just yet.